Jenkinsfile

```diff
@@ -520,7 +520,7 @@ def testGo(prefix, packagesToTest) {
       }
 
       println "Running go vet for ${pkg}"
-      sh "go vet ${pkg}"
+      sh "go vet ${pkg} || (ERR=\$? && echo \"go vet failed with error code \$ERR\" && exit \$ERR)"
```

Comment:
I think this can be `(ERR=\$? echo \"...\")`, although if it works the current way you can leave it.

Reply:
Not 100% sure about this, it still needs to exit so it has to be shared across 2 commands.

Reply:
True, fair enough.